### PR TITLE
Added a landing page to the site

### PR DESCRIPTION
**Pull request by aidan-brown**

### CODE REVIEW

**Comment by rmarks6767** [comment]
70 files 😳😳😳 at this hour

File: frontend/src/Components/App/App.jsx [hunk 1]
```diff
@@ -1,26 +1,37 @@
...
 const App = () => {
+  const streamDate = new Date('April 8, 2022 19:00:00');
   const [selectedPlayer, setSelectedPlayer] = useState('');
+  const [remainingTime, setRemainingTime] = useState(streamDate - Date.now());
+  const [countdown, setCountdown] = useState(msToTime(remainingTime));
+
+  useEffect(() => {
+    setTimeout(() => {
+      setRemainingTime(streamDate - Date.now());
+      setCountdown(msToTime(remainingTime));
```
Comment (rmarks6767) [inline frontend/src/Components/App/App.jsx]: Why do you need two countdown variables? `setCountdown` is redundant

**Comment by aidan-brown** [comment]
you know it

**Review by rmarks6767 — APPROVED** [review]
Looks good other than non blocking unneeded variable

File: frontend/src/Components/App/App.jsx (hunk 1 above)
Comment (aidan-brown) [reply frontend/src/Components/App/App.jsx]: because im dumb and didn't remove that

**Pull request merged**